jkube-kit/parent/pom.xml

```diff
@@ -57,7 +57,7 @@
     <version.json-path-assert>2.2.0</version.json-path-assert>
     <version.jsr305>3.0.2</version.jsr305>
     <version.JUnitParams>1.0.5</version.JUnitParams>
-    <version.kubernetes-client>4.6.2</version.kubernetes-client>
+    <version.kubernetes-client>4.7.1</version.kubernetes-client>
```

Comment:
Shall we upgrade to 4.8.0 instead?

Reply:
As JKube requires CQ to upgrade a dependency, I'd merge this one once the rest of the comments are fine.
Then do the upgrade to 4.8.0 in the scope of another issue.
Else we'll be stuck forever.